### ISSUE TITLE
Training on new data

### ISSUE THREAD

**Issue by junaidamk** [jiesutd/NCRFpp#110]
https://github.com/jiesutd/NCRFpp/issues/104#issuecomment-486535264

**Comment by junaidamk** [comment]
hello.........if you feel free to reply I need some more clarification regarding my task..
i have trained with pretrained fasttext embedding, which also gives very little improvement in accuracy?
what i can try to improve the model accuracy? any suggesion please......

**Comment by jiesutd** [comment]
Your loss is still very large when the training finished. You may need to tune the hyperparameters e.g. learning rate, iteration number, etc.

**Comment by junaidamk** [comment]
https://github.com/jiesutd/NCRFpp/issues/104#issuecomment-480272183
here also my loss is high..but you said that everything is alright.. ???
